src/inspector/models/abstractinspectormodel.cpp

```diff
@@ -814,7 +793,7 @@ void AbstractInspectorModel::beginCommand(const muse::TranslatableString& action
 
     //! NOTE prevents unnecessary updating of properties
     //! after changing their values in the inspector
-    m_updatePropertiesAllowed = false;
+    m_shouldUpdateOnScoreChange = false;
```

Comment:
@RomanPudashkin It seems we never set it to `true` again. Isn't that a problem?

Reply:
Yeah, that seems a good solution indeed. I could verify that that doesn't break the purpose, i.e. updates from "inside" the model are still ignored.